lib/preview_web/router.ex

```diff
@@ -29,18 +33,13 @@ defmodule PreviewWeb.Router do
     live "/preview/:package/:version/show/*filename", PreviewLive, :index
   end
 
-  # Other scopes may use custom stacks.
-  # scope "/api", PreviewWeb do
-  #   pipe_through :api
-  # end
-
-  # Enables LiveDashboard only for development
-  #
-  # If you want to use the LiveDashboard in production, you should put
-  # it behind authentication and allow only admins to access it.
-  # If your application does not have an admins-only section yet,
-  # you can use Plug.BasicAuth to set up some basic authentication
-  # as long as you are also using SSL (which you should anyway).
+  scope "/", PreviewWeb do
+    pipe_through :raw
+
+    get "/preview/:package/raw/*filename", PackageController, :raw
+    get "/preview/:package/:version/raw/*filename", PackageController, :raw
```

Comment:
these routes match the LiveView ones but in hindsight perhaps something like this would be easier to work with?

```
/preview/:package/latest/*filename
/preview/:package/:version/*filename
```

I think either way is fine but it's last call to make this change. :)